test/e2e/mpi_job_test.go

```diff
@@ -164,8 +165,43 @@ var _ = ginkgo.Describe("MPIJob", func() {
 				mpiJob := createJobAndWaitForCompletion(mpiJob)
 				expectConditionToBeTrue(mpiJob, kubeflow.JobSucceeded)
 			})
-		})
 
+			ginkgo.It("should not be updated when managed externaly, only created", func() {
+				mpiJob.Spec.RunPolicy.ManagedBy = ptr.To(kubeflow.MultiKueueController)
+				ctx := context.Background()
+				mpiJob = createJob(ctx, mpiJob)
+
+				gomega.Consistently(func() error {
+					updatedJob, err := mpiClient.KubeflowV2beta1().MPIJobs(mpiJob.Namespace).Get(ctx, mpiJob.Name, metav1.GetOptions{})
+					if err != nil {
+						return err
+					}
+					mpiJob = updatedJob
+					return nil
+				}, 2*time.Second, waitInterval).Should(gomega.Succeed())
+
+				// job should be created, but status should not be updated neither for create nor for any other status
+				condition := getJobCondition(mpiJob, kubeflow.JobCreated)
+				gomega.Expect(condition).To(gomega.BeNil())
+				condition = getJobCondition(mpiJob, kubeflow.JobSucceeded)
+				gomega.Expect(condition).To(gomega.BeNil())
+				launcherPods, err := getLauncherPods(ctx, mpiJob)
```

Comment:
Could you verify if the Launcher Job doesn't exist instead of the Launcher Pod?

Reply:
done